### PR TITLE
chore(codegen): include default value comment for service id

### PR DESCRIPTION
**Pull request by eduardomourar**
### Issue
N/A

### Description
This will include the default value as comments for the service identifier.

### Testing
Yes, just locally

### Additional context
Related to PR #4139. It will allow us to use that value during the documentation generation as shown here: https://github.com/aws/aws-sdk-js-v3/blob/cb102a6af6772a982d836eb9804ffb38293fa4ef/packages/service-client-documentation-generator/src/sdk-client-rename-project.ts#L26-L34

---
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/AddAwsRuntimeConfig.java [hunk 1]
```diff
@@ -77,7 +77,10 @@ public void addConfigInterfaceFields(
             TypeScriptWriter writer
     ) {
         if (isAwsService(settings, model)) {
-            writer.writeDocs("Unique service identifier.\n@internal")
+            ServiceShape service = settings.getService(model);
+            String sdkId = service.getTrait(ServiceTrait.class).map(ServiceTrait::getSdkId)
+                    .orElse("");
+            writer.writeDocs(String.format("Unique service identifier.%n@internal%n@default \"%s\"", sdkId))
```
Comment (kuhe) [inline codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/AddAwsRuntimeConfig.java:83]: Can you run the codegen (`yarn generate-clients`) for this and commit it, or want us to do it?

**Review by kuhe — APPROVED** [review]

File: codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/AddAwsRuntimeConfig.java (hunk 1 above)
Comment (eduardomourar) [reply codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/AddAwsRuntimeConfig.java:83]: I though that was done in a separate process.

Comment (eduardomourar) [reply codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/AddAwsRuntimeConfig.java:83]: I ran the generate-clients script now.

Comment (eduardomourar) [reply codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/AddAwsRuntimeConfig.java:83]: i reverted the clients update in order to get this merged

**Pull request closed without merging**